src/config.ts

```diff
@@ -63,6 +63,10 @@ const configSchema = Type.Object(
      * E.g. Date will be of Type String when enabled.
      */
     useJsonTypes: Type.Boolean({ default: false }),
+    /**
+     * Wether to add ".js" extensions to src file imports to support nodenext module resolution
+     */
+    explicitFileExtensions: Type.Boolean({ default: false }),
```

Comment:
Is there a possible scenario where this should be configurable? like I now want this to be '.mjs' but for my other project I need this to be '.ts' or just '.js'? Is this a thing or will it just never happen?

Reply:
Unsure if its important but might as well make it configurable. I've pushed an update for this